Rename APIs to match Mojang's naming

modmuss50 opened the pull request:
# API Renames

#### `net/fabricmc/fabric/api/biome/v1/BiomeModificationContext$EffectsContext`
- `setFoliageColor` → `setFoliageColorOverride`
- `clearFoliageColor` → `clearFoliageColorOverride`
- `setDryFoliageColor` → `setDryFoliageColorOverride`
- `clearDryFoliageColor` → `clearDryFoliageColorOverride`
- `setGrassColor` → `setGrassColorOverride`
- `clearGrassColor` → `clearGrassColorOverride`

#### `net/fabricmc/fabric/api/biome/v1/BiomeModificationContext$SpawnSettingsContext` → `BiomeModificationContext$MobSpawnSettingsContext`
- `setCreatureSpawnProbability` → `setCreatureGenerationProbability`
- `getSpawnEntries` → `getMobs`
- `setSpawnCost` → `addMobCharge`
- `clearSpawnCost` → `clearMobCharge`

#### `net/fabricmc/fabric/api/biome/v1/BiomeModificationContext`
- `getSpawnSettings` → `getMobSpawnSettings`

#### `net/fabricmc/fabric/api/biome/v1/BiomeSelectionContext`
- `getBiomeRegistryEntry` → `getBiomeHolder`

#### `net/fabricmc/fabric/api/client/command/v2/ClientCommandManager` → `ClientCommands`

#### `net/fabricmc/fabric/api/client/command/v2/FabricClientCommandSource`
- `getWorld` → `getLevel`

#### `net/fabricmc/fabric/api/client/datagen/v1/builder/SoundTypeBuilder$EntryBuilder` → `SoundTypeBuilder$RegistrationBuilder`

#### `net/fabricmc/fabric/api/client/datagen/v1/builder/SoundTypeBuilder`
- `category` → `source`

#### `net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientTickEvents$EndWorldTick` → `ClientTickEvents$EndLevelTick`

#### `net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientTickEvents$StartWorldTick` → `ClientTickEvents$StartLevelTick`

#### `net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientTickEvents`
- `START_WORLD_TICK` → `START_LEVEL_TICK`
- `END_WORLD_TICK` → `END_LEVEL_TICK`

#### `net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientWorldEvents$AfterClientWorldChange` → `ClientLevelEvents$AfterClientLevelChange`
- `afterWorldChange` → `afterLevelChange`

#### `net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientWorldEvents` → `ClientLevelEvents`
- `AFTER_CLIENT_WORLD_CHANGE` → `AFTER_CLIENT_LEVEL_CHANGE`

#### `net/fabricmc/fabric/api/client/itemgroup/v1/FabricCreativeInventoryScreen` → `net/fabricmc/fabric/api/client/creativetab/v1/FabricCreativeModeInventoryScreen`
- `getItemGroupsOnPage` → `getTabsOnPage`
- `getSelectedItemGroup` → `getSelectedTab`
- `setSelectedItemGroup` → `setSelectedTab`

#### `net/fabricmc/fabric/api/client/itemgroup/v1/package-info` → `net/fabricmc/fabric/api/client/creativetab/v1/package-info`

#### `net/fabricmc/fabric/api/client/keybinding/v1/KeyBindingHelper` → `net/fabricmc/fabric/api/client/keymapping/v1/KeyMappingHelper`
- `registerKeyBinding` → `registerKeyMapping`

#### `net/fabricmc/fabric/api/client/keybinding/v1/package-info` → `net/fabricmc/fabric/api/client/keymapping/v1/package-info`

#### `net/fabricmc/fabric/api/client/model/loading/v1/FabricBakedModelManager` → `FabricModelManager`

#### `net/fabricmc/fabric/api/client/model/loading/v1/ModelModifier$AfterBakeItem$Context`
- `bakeContext` → `bakingContext`

#### `net/fabricmc/fabric/api/client/model/loading/v1/ModelModifier$BeforeBakeItem$Context`
- `bakeContext` → `bakingContext`

#### `net/fabricmc/fabric/api/client/model/loading/v1/wrapper/WrapperUnbakedGroupedBlockStateModel` → `WrapperUnbakedRootBlockStateModel`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SConfigurationChannelEvents$Register` → `ServerboundConfigurationChannelEvents$Register`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SConfigurationChannelEvents$Unregister` → `ServerboundConfigurationChannelEvents$Unregister`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SConfigurationChannelEvents` → `ServerboundConfigurationChannelEvents`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SPlayChannelEvents$Register` → `ServerboundPlayChannelEvents$Register`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SPlayChannelEvents$Unregister` → `ServerboundPlayChannelEvents$Unregister`

#### `net/fabricmc/fabric/api/client/networking/v1/C2SPlayChannelEvents` → `ServerboundPlayChannelEvents`

#### `net/fabricmc/fabric/api/client/networking/v1/ClientConfigurationNetworking$Context`
- `networkHandler` → `packetListener`

#### `net/fabricmc/fabric/api/client/networking/v1/ClientPlayNetworking`
- `createC2SPacket` → `createServerboundPacket`

#### `net/fabricmc/fabric/api/client/particle/v1/FabricSpriteProvider` → `FabricSpriteSet`

#### `net/fabricmc/fabric/api/client/particle/v1/ParticleFactoryRegistry$PendingParticleFactory` → `ParticleProviderRegistry$PendingParticleProvider`

#### `net/fabricmc/fabric/api/client/particle/v1/ParticleFactoryRegistry` → `ParticleProviderRegistry`

#### `net/fabricmc/fabric/api/client/particle/v1/ParticleRenderEvents$AllowBlockDustTint` → `ParticleRenderEvents$AllowTerrainParticleTint`
- `allowBlockDustTint` → `allowTerrainParticleTint`

#### `net/fabricmc/fabric/api/client/particle/v1/ParticleRenderEvents`
- `ALLOW_BLOCK_DUST_TINT` → `ALLOW_TERRAIN_PARTICLE_TINT`

#### `net/fabricmc/fabric/api/client/particle/v1/ParticleRendererRegistry` → `ParticleGroupRegistry`
- `getParticleTextureSheet` → `getParticleRenderType`

#### `net/fabricmc/fabric/api/client/rendering/v1/AtlasSourceRegistry` → `SpriteSourceRegistry`

#### `net/fabricmc/fabric/api/client/rendering/v1/BlockRenderLayerMap` → `ChunkSectionLayerMap`

#### `net/fabricmc/fabric/api/client/rendering/v1/DrawItemStackOverlayCallback` → `RenderItemDecorationsCallback`
- `onDrawItemStackOverlay` → `onRenderItemDecorations`

#### `net/fabricmc/fabric/api/client/rendering/v1/EntityModelLayerRegistry$TexturedEquipmentModelDataProvider` → `ModelLayerRegistry$TexturedArmorModelSetProvider`
- `createEquipmentModelData` → `createArmorModelSet`

#### `net/fabricmc/fabric/api/client/rendering/v1/EntityModelLayerRegistry$TexturedModelDataProvider` → `ModelLayerRegistry$TexturedLayerDefinitionProvider`
- `createModelData` → `createLayerDefinition`

#### `net/fabricmc/fabric/api/client/rendering/v1/EntityModelLayerRegistry` → `ModelLayerRegistry`
- `registerEquipmentModelLayers` → `registerArmorModelLayers`

#### `net/fabricmc/fabric/api/client/rendering/v1/LivingEntityFeatureRendererRegistrationCallback$RegistrationHelper` → `LivingEntityRenderLayerRegistrationCallback$RegistrationHelper`

#### `net/fabricmc/fabric/api/client/rendering/v1/LivingEntityFeatureRendererRegistrationCallback` → `LivingEntityRenderLayerRegistrationCallback`
- `registerRenderers` → `registerLayers`

#### `net/fabricmc/fabric/api/client/rendering/v1/SpecialGuiElementRegistry$Context` → `PictureInPictureRendererRegistry$Context`
- `vertexConsumers` → `bufferSource`
- `client` → `minecraft`
- `orderedRenderCommandQueue` → `submitNodeCollector`

#### `net/fabricmc/fabric/api/client/rendering/v1/SpecialGuiElementRegistry$Factory` → `PictureInPictureRendererRegistry$Factory`
- `createSpecialRenderer` → `createRenderer`

#### `net/fabricmc/fabric/api/client/rendering/v1/SpecialGuiElementRegistry` → `PictureInPictureRendererRegistry`

#### `net/fabricmc/fabric/api/client/rendering/v1/TooltipComponentCallback` → `ClientTooltipComponentCallback`
- `getComponent` → `getClientComponent`

#### `net/fabricmc/fabric/api/client/rendering/v1/hud/VanillaHudElements`
- `STATUS_EFFECTS` → `MOB_EFFECTS`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/AbstractWorldRenderContext` → `net/fabricmc/fabric/api/client/rendering/v1/level/AbstractLevelRenderContext`
- `worldRenderer` → `levelRenderer`
- `worldState` → `levelState`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldExtractionContext` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelExtractionContext`
- `world` → `level`
- `tickCounter` → `deltaTracker`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderContext` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderContext`
- `commandQueue` → `submitNodeCollector`
- `matrices` → `poseStack`
- `consumers` → `bufferSource`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$AfterBlockOutlineExtraction` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$AfterBlockOutlineExtraction`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$AfterEntities` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$AfterEntities`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$BeforeBlockOutline` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$BeforeBlockOutline`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$BeforeEntities` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$BeforeEntities`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$BeforeTranslucent` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$BeforeTranslucent`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$DebugRender` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$DebugRender`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$EndExtraction` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$EndExtraction`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$EndMain` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$EndMain`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents$StartMain` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents$StartMain`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldRenderEvents` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelRenderEvents`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/WorldTerrainRenderContext` → `net/fabricmc/fabric/api/client/rendering/v1/level/LevelTerrainRenderContext`
- `sectionState` → `sectionsToRender`

#### `net/fabricmc/fabric/api/client/rendering/v1/world/package-info` → `net/fabricmc/fabric/api/client/rendering/v1/level/package-info`

#### `net/fabricmc/fabric/api/client/screen/v1/Screens`
- `getButtons` → `getWidgets`
- `getTextRenderer` → `getFont`
- `getClient` → `getMinecraft`

#### `net/fabricmc/fabric/api/command/v2/FabricEntitySelectorReader` → `FabricEntitySelectorParser`

#### `net/fabricmc/fabric/api/datagen/v1/FabricDataOutput` → `FabricPackOutput`

#### `net/fabricmc/fabric/api/datagen/v1/loot/FabricBlockLootTableGenerator` → `FabricBlockLootSubProvider`

#### `net/fabricmc/fabric/api/datagen/v1/loot/FabricEntityLootTableGenerator` → `FabricEntityLootSubProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricAdvancementProvider`
- `pathResolver` → `pathProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricBlockLootTableProvider` → `FabricBlockLootSubProvider`
- `registryLookupFuture` → `registriesFuture`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricCodecDataProvider`
- `pathResolver` → `pathProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricDynamicRegistryProvider$RegistryEntries`
- `entries` → `resources`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricDynamicRegistryProvider`
- `writeRegistryEntries` → `writeHolders`
- `lambda$writeRegistryEntries$0` → `lambda$writeHolders$0`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricEntityLootTableProvider` → `FabricEntityLootSubProvider`
- `registryLookupFuture` → `registriesFuture`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricLanguageProvider`
- `dataOutput` → `packOutput`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricLootTableProvider` → `FabricLootTableSubProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricProvidedTagBuilder` → `FabricTagAppender`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$AliasGroupBuilder` → `FabricTagsProvider$AliasGroupBuilder`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$BlockEntityTypeTagProvider` → `FabricTagsProvider$BlockEntityTypeTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$BlockTagProvider` → `FabricTagsProvider$BlockTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$EntityTypeTagProvider` → `FabricTagsProvider$EntityTypeTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$FabricValueLookupTagProvider` → `FabricTagsProvider$FabricIntrinsicHolderTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$FluidTagProvider` → `FabricTagsProvider$FluidTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider$ItemTagProvider` → `FabricTagsProvider$ItemTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider` → `FabricTagsProvider`

#### `net/fabricmc/fabric/api/datagen/v1/provider/SimpleFabricLootTableProvider` → `SimpleFabricLootTableSubProvider`
- `registryLookup` → `registryLookupFuture`
- `contextType` → `contextParamSet`

#### `net/fabricmc/fabric/api/datagen/v1/recipe/FabricRecipeExporter` → `FabricRecipeOutput`

#### `net/fabricmc/fabric/api/entity/FakePlayer$FakePlayerKey`
- `world` → `level`

- `world` → `level`

#### `net/fabricmc/fabric/api/entity/event/v1/ServerEntityWorldChangeEvents$AfterEntityChange` → `ServerEntityLevelChangeEvents$AfterEntityChange`
- `afterChangeWorld` → `afterChangeLevel`

#### `net/fabricmc/fabric/api/entity/event/v1/ServerEntityWorldChangeEvents$AfterPlayerChange` → `ServerEntityLevelChangeEvents$AfterPlayerChange`
- `afterChangeWorld` → `afterChangeLevel`

#### `net/fabricmc/fabric/api/entity/event/v1/ServerEntityWorldChangeEvents` → `ServerEntityLevelChangeEvents`
- `AFTER_ENTITY_CHANGE_WORLD` → `AFTER_ENTITY_CHANGE_LEVEL`
- `AFTER_PLAYER_CHANGE_WORLD` → `AFTER_PLAYER_CHANGE_LEVEL`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerChunkEvents$LevelTypeChange` → `ServerChunkEvents$FullChunkStatusChange`
- `onChunkLevelTypeChange` → `onFullChunkStatusChange`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerChunkEvents`
- `CHUNK_LEVEL_TYPE_CHANGE` → `FULL_CHUNK_STATUS_CHANGE`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerTickEvents$EndWorldTick` → `ServerTickEvents$EndLevelTick`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerTickEvents$StartWorldTick` → `ServerTickEvents$StartLevelTick`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerTickEvents`
- `START_WORLD_TICK` → `START_LEVEL_TICK`
- `END_WORLD_TICK` → `END_LEVEL_TICK`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerWorldEvents$Load` → `ServerLevelEvents$Load`
- `onWorldLoad` → `onLevelLoad`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerWorldEvents$Unload` → `ServerLevelEvents$Unload`
- `onWorldUnload` → `onLevelUnload`

#### `net/fabricmc/fabric/api/event/lifecycle/v1/ServerWorldEvents` → `ServerLevelEvents`

#### `net/fabricmc/fabric/api/event/registry/DynamicRegistryView`
- `asDynamicRegistryManager` → `asRegistryAccess`

#### `net/fabricmc/fabric/api/event/registry/FabricRegistryBuilder`
- `createSimple` → `create`

#### `net/fabricmc/fabric/api/gamerule/v1/FabricGameRuleVisitor` → `FabricGameRuleTypeVisitor`

#### `net/fabricmc/fabric/api/item/v1/ComponentTooltipAppenderRegistry` → `ItemComponentTooltipProviderRegistry`

#### `net/fabricmc/fabric/api/item/v1/CustomDamageHandler`
- `damage` → `hurtAndBreak`

#### `net/fabricmc/fabric/api/item/v1/EquipmentSlotProvider`
- `getPreferredEquipmentSlot` → `getEquipmentSlotForItem`

#### `net/fabricmc/fabric/api/item/v1/FabricItem$Settings` → `FabricItem$Properties`

#### `net/fabricmc/fabric/api/item/v1/FabricItem`
- `getRecipeRemainder` → `getCraftingRemainder`

#### `net/fabricmc/fabric/api/item/v1/FabricItemStack`
- `getRecipeRemainder` → `getCraftingRemainder`

#### `net/fabricmc/fabric/api/itemgroup/v1/FabricItemGroup` → `net/fabricmc/fabric/api/creativetab/v1/FabricCreativeModeTab`

#### `net/fabricmc/fabric/api/itemgroup/v1/FabricItemGroupEntries` → `net/fabricmc/fabric/api/creativetab/v1/FabricCreativeModeTabOutput`
- `addAfter` → `insertAfter`
- `addBefore` → `insertBefore`

#### `net/fabricmc/fabric/api/itemgroup/v1/ItemGroupEvents$ModifyEntries` → `net/fabricmc/fabric/api/creativetab/v1/CreativeModeTabEvents$ModifyOutput`
- `modifyEntries` → `modifyOutput`

#### `net/fabricmc/fabric/api/itemgroup/v1/ItemGroupEvents$ModifyEntriesAll` → `net/fabricmc/fabric/api/creativetab/v1/CreativeModeTabEvents$ModifyOutputAll`
- `modifyEntries` → `modifyOutput`

#### `net/fabricmc/fabric/api/itemgroup/v1/ItemGroupEvents` → `net/fabricmc/fabric/api/creativetab/v1/CreativeModeTabEvents`
- `modifyEntriesEvent` → `modifyOutputEvent`

- `MODIFY_ENTRIES_ALL` → `MODIFY_OUTPUT_ALL`

#### `net/fabricmc/fabric/api/itemgroup/v1/package-info` → `net/fabricmc/fabric/api/creativetab/v1/package-info`

#### `net/fabricmc/fabric/api/lookup/v1/block/BlockApiCache`
- `getWorld` → `getLevel`

#### `net/fabricmc/fabric/api/loot/v3/FabricLootPoolBuilder`
- `with` → `add`
- `conditionally` → `when`

#### `net/fabricmc/fabric/api/networking/v1/FabricServerConfigurationNetworkHandler` → `FabricServerConfigurationPacketListenerImpl`

#### `net/fabricmc/fabric/api/networking/v1/PacketByteBufs` → `FriendlyByteBufs`
- `EMPTY_PACKET_BYTE_BUF` → `EMPTY_FRIENDLY_BYTE_BUF`

#### `net/fabricmc/fabric/api/networking/v1/PayloadTypeRegistry`
- `configurationC2S` → `serverboundConfiguration`
- `configurationS2C` → `clientboundConfiguration`
- `playC2S` → `serverboundPlay`
- `playS2C` → `clientboundPlay`

#### `net/fabricmc/fabric/api/networking/v1/PlayerLookup`
- `world` → `level`

#### `net/fabricmc/fabric/api/networking/v1/S2CConfigurationChannelEvents$Register` → `ClientboundConfigurationChannelEvents$Register`

#### `net/fabricmc/fabric/api/networking/v1/S2CConfigurationChannelEvents$Unregister` → `ClientboundConfigurationChannelEvents$Unregister`

#### `net/fabricmc/fabric/api/networking/v1/S2CConfigurationChannelEvents` → `ClientboundConfigurationChannelEvents`

#### `net/fabricmc/fabric/api/networking/v1/S2CPlayChannelEvents$Register` → `ClientboundPlayChannelEvents$Register`

#### `net/fabricmc/fabric/api/networking/v1/S2CPlayChannelEvents$Unregister` → `ClientboundPlayChannelEvents$Unregister`

#### `net/fabricmc/fabric/api/networking/v1/S2CPlayChannelEvents` → `ClientboundPlayChannelEvents`

#### `net/fabricmc/fabric/api/networking/v1/ServerConfigurationNetworking$Context`
- `networkHandler` → `packetListener`

#### `net/fabricmc/fabric/api/networking/v1/ServerConfigurationNetworking`
- `createS2CPacket` → `createClientboundPacket`

#### `net/fabricmc/fabric/api/networking/v1/ServerPlayNetworking`
- `createS2CPacket` → `createClientboundPacket`

#### `net/fabricmc/fabric/api/object/builder/v1/block/entity/FabricBlockEntityType`
- `addSupportedBlock` → `addValidBlock`

#### `net/fabricmc/fabric/api/object/builder/v1/block/type/BlockSetTypeBuilder`
- `soundGroup` → `soundType`

- `soundGroup` → `soundType`

#### `net/fabricmc/fabric/api/object/builder/v1/block/type/WoodTypeBuilder`
- `soundGroup` → `soundType`
- `hangingSignSoundGroup` → `hangingSignSoundType`

- `soundGroup` → `soundType`
- `hangingSignSoundGroup` → `hangingSignSoundType`

#### `net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityType$Builder$Mob`
- `spawnRestriction` → `spawnPlacement`

#### `net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder$Living`
- `spawnGroup` → `mobCategory`

#### `net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder$Mob`
- `spawnGroup` → `mobCategory`
- `spawnRestriction` → `spawnPlacement`

- `spawnLocation` → `spawnPlacementType`
- `restrictionHeightmap` → `placementHeightmap`

#### `net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder`
- `spawnGroup` → `mobCategory`

- `spawnGroup` → `mobCategory`

#### `net/fabricmc/fabric/api/object/builder/v1/entity/FabricTrackedDataRegistry` → `FabricEntityDataRegistry`

#### `net/fabricmc/fabric/api/object/builder/v1/world/poi/PointOfInterestHelper` → `PoiHelper`

#### `net/fabricmc/fabric/api/particle/v1/FabricBlockStateParticleEffect` → `FabricBlockParticleOption`

#### `net/fabricmc/fabric/api/recipe/v1/FabricRecipeManager` → `FabricRecipeAccess`

#### `net/fabricmc/fabric/api/recipe/v1/FabricServerRecipeManager` → `FabricRecipeManager`

#### `net/fabricmc/fabric/api/recipe/v1/ingredient/CustomIngredient`
- `getMatchingItems` → `items`
- `toDisplay` → `display`

#### `net/fabricmc/fabric/api/recipe/v1/ingredient/CustomIngredientSerializer`
- `getPacketCodec` → `getStreamCodec`

#### `net/fabricmc/fabric/api/registry/CompostingChanceRegistry` → `CompostableRegistry`

#### `net/fabricmc/fabric/api/registry/FabricBrewingRecipeRegistryBuilder$BuildCallback` → `FabricPotionBrewingBuilder$BuildCallback`

#### `net/fabricmc/fabric/api/registry/FabricBrewingRecipeRegistryBuilder` → `FabricPotionBrewingBuilder`

#### `net/fabricmc/fabric/api/registry/FlammableBlockRegistry$Entry`
- `getBurnChance` → `getIgniteOdds`
- `getSpreadChance` → `getBurnOdds`

- `burn` → `igniteOdds`
- `spread` → `burnOdds`

#### `net/fabricmc/fabric/api/registry/FuelRegistryEvents$BuildCallback` → `FuelValueEvents$BuildCallback`

#### `net/fabricmc/fabric/api/registry/FuelRegistryEvents$Context` → `FuelValueEvents$Context`

#### `net/fabricmc/fabric/api/registry/FuelRegistryEvents$ExclusionsCallback` → `FuelValueEvents$ExclusionsCallback`

#### `net/fabricmc/fabric/api/registry/FuelRegistryEvents` → `FuelValueEvents`

#### `net/fabricmc/fabric/api/registry/LandPathNodeTypesRegistry$DynamicPathNodeTypeProvider` → `LandPathTypeRegistry$DynamicPathTypeProvider`
- `getPathNodeType` → `getPathType`

#### `net/fabricmc/fabric/api/registry/LandPathNodeTypesRegistry$PathNodeTypeProvider` → `LandPathTypeRegistry$PathTypeProvider`

#### `net/fabricmc/fabric/api/registry/LandPathNodeTypesRegistry$StaticPathNodeTypeProvider` → `LandPathTypeRegistry$StaticPathTypeProvider`
- `getPathNodeType` → `getPathType`

#### `net/fabricmc/fabric/api/registry/LandPathNodeTypesRegistry` → `LandPathTypeRegistry`
- `getPathNodeType` → `getPathType`
- `getPathNodeTypeProvider` → `getPathTypeProvider`

- `NODE_TYPES` → `PATH_TYPES`

#### `net/fabricmc/fabric/api/registry/OxidizableBlocksRegistry`
- `registerOxidizableBlockPair` → `registerNextStage`
- `registerWaxableBlockPair` → `registerWaxable`
- `registerCopperBlockSet` → `registerWeatheringCopperBlocks`

#### `net/fabricmc/fabric/api/registry/SculkSensorFrequencyRegistry` → `VibrationFrequencyRegistry`

#### `net/fabricmc/fabric/api/registry/VillagerInteractionRegistries`
- `registerCollectable` → `registerGatherableItem`

#### `net/fabricmc/fabric/api/renderer/v1/mesh/MutableQuadView`
- `renderLayer` → `chunkLayer`
- `glint` → `foilType`

#### `net/fabricmc/fabric/api/renderer/v1/mesh/QuadEmitter`
- `renderLayer` → `chunkLayer`
- `glint` → `foilType`

#### `net/fabricmc/fabric/api/renderer/v1/mesh/QuadView`
- `renderLayer` → `chunkLayer`
- `glint` → `foilType`

#### `net/fabricmc/fabric/api/renderer/v1/model/FabricBlockModels` → `FabricBlockModelShaper`
- `getModelParticleSprite` → `getParticleIcon`

#### `net/fabricmc/fabric/api/renderer/v1/model/FabricBlockStateModel`
- `particleSprite` → `particleIcon`

#### `net/fabricmc/fabric/api/renderer/v1/model/MeshBakedGeometry` → `MeshQuadCollection`

#### `net/fabricmc/fabric/api/renderer/v1/model/ModelBakeSettingsHelper` → `ModelStateHelper`

#### `net/fabricmc/fabric/api/renderer/v1/render/BlockVertexConsumerProvider` → `BlockMultiBufferSource`

#### `net/fabricmc/fabric/api/renderer/v1/render/FabricBlockModelRenderer` → `FabricModelBlockRenderer`

#### `net/fabricmc/fabric/api/renderer/v1/render/FabricBlockRenderManager` → `FabricBlockRenderDispatcher`

#### `net/fabricmc/fabric/api/renderer/v1/render/FabricRenderCommandQueue` → `FabricOrderedSubmitNodeCollector`

#### `net/fabricmc/fabric/api/renderer/v1/render/RenderLayerHelper` → `ChunkSectionLayerHelper`

#### `net/fabricmc/fabric/api/renderer/v1/sprite/FabricErrorCollectingSpriteGetter` → `FabricSpriteGetter`

#### `net/fabricmc/fabric/api/renderer/v1/sprite/FabricSpriteAtlasTexture` → `FabricPreparations`

#### `net/fabricmc/fabric/api/renderer/v1/sprite/FabricStitchResult` → `FabricTextureAtlas`

#### `net/fabricmc/fabric/api/resource/v1/DataResourceLoader`
- `registerReloader` → `registerReloadListener`

#### `net/fabricmc/fabric/api/resource/v1/ResourceLoader`
- `registerReloader` → `registerReloadListener`
- `addReloaderOrdering` → `addListenerOrdering`

- `RELOADER_REGISTRY_LOOKUP_KEY` → `REGISTRY_LOOKUP_KEY`
- `RELOADER_FEATURE_SET_KEY` → `FEATURE_FLAG_SET_KEY`

#### `net/fabricmc/fabric/api/resource/v1/reloader/ResourceReloaderKeys$Client`
- `BLOCK_ENTITY_RENDERERS` → `BLOCK_ENTITY_RENDER_DISPATCHER`
- `BLOCK_RENDER_MANAGER` → `BLOCK_RENDER_DISPATCHER`
- `CLOUD_CELLS` → `CLOUD_RENDERER`
- `EQUIPMENT_MODELS` → `EQUIPMENT_ASSETS`
- `ENTITY_RENDERERS` → `ENTITY_RENDER_DISPATCHER`
- `DRY_FOLIAGE_COLORMAP` → `DRY_FOLIAGE_COLOR`
- `FOLIAGE_COLORMAP` → `FOLIAGE_COLOR`
- `GRASS_COLORMAP` → `GRASS_COLOR`
- `WAYPOINT_STYLE_ASSETS` → `WAYPOINT_STYLE`

#### `net/fabricmc/fabric/api/resource/v1/reloader/SimpleResourceReloader` → `SimpleReloadListener`

#### `net/fabricmc/fabric/api/screenhandler/v1/ExtendedScreenHandlerFactory` → `net/fabricmc/fabric/api/menu/v1/ExtendedMenuProvider`

#### `net/fabricmc/fabric/api/screenhandler/v1/ExtendedScreenHandlerType$ExtendedFactory` → `net/fabricmc/fabric/api/menu/v1/ExtendedMenuType$ExtendedFactory`

#### `net/fabricmc/fabric/api/screenhandler/v1/ExtendedScreenHandlerType` → `net/fabricmc/fabric/api/menu/v1/ExtendedMenuType`
- `getPacketCodec` → `getStreamCodec`

- `packetCodec` → `streamCodec`

#### `net/fabricmc/fabric/api/screenhandler/v1/FabricScreenHandlerFactory` → `net/fabricmc/fabric/api/menu/v1/FabricMenuProvider`

#### `net/fabricmc/fabric/api/screenhandler/v1/package-info` → `net/fabricmc/fabric/api/menu/v1/package-info`

#### `net/fabricmc/fabric/api/serialization/v1/view/FabricReadView` → `net/fabricmc/fabric/api/serialization/v1/value/FabricValueInput`
- `keys` → `keySet`

#### `net/fabricmc/fabric/api/serialization/v1/view/FabricWriteView` → `net/fabricmc/fabric/api/serialization/v1/value/FabricValueOutput`

#### `net/fabricmc/fabric/api/serialization/v1/view/package-info` → `net/fabricmc/fabric/api/serialization/v1/value/package-info`

#### `net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariant`
- `getRegistryEntry` → `typeHolder`
- `withComponentChanges` → `withComponents`

#### `net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariantAttributeHandler`
- `getLuminance` → `getLightEmission`

#### `net/fabricmc/fabric/api/transfer/v1/fluid/base/SingleFluidStorage`
- `readData` → `readValue`
- `writeData` → `writeValue`

#### `net/fabricmc/fabric/api/transfer/v1/item/InventoryStorage` → `ContainerStorage`

#### `net/fabricmc/fabric/api/transfer/v1/item/ItemVariant`
- `getRegistryEntry` → `typeHolder`
- `withComponentChanges` → `withComponents`

#### `net/fabricmc/fabric/api/transfer/v1/item/base/SingleItemStorage`
- `readData` → `readValue`
- `writeData` → `writeValue`

#### `net/fabricmc/fabric/api/transfer/v1/storage/StorageUtil`
- `calculateComparatorOutput` → `getRedstoneSignal`

#### `net/fabricmc/fabric/api/transfer/v1/storage/TransferVariant`
- `getComponents` → `getComponentsPatch`
- `getComponentMap` → `getComponents`
- `withComponentChanges` → `withComponents`

#### `net/fabricmc/fabric/api/transfer/v1/storage/base/SingleVariantStorage`
- `readData` → `readValue`
- `writeData` → `writeValue`

Tenneb22 committed:
Change include in settings.gradle from 'fabric-screen-handler-api-v1' to 'fabric-menu-api-v1'

Tenneb22 committed:
Rename fabric-screen-handler-api-v1 directory to fabric-menu-api-v1

Tenneb22 committed:
Rename fabric-screen-handler-api-v1-version to fabric-menu-api-v1-version in gradle.properties
(not sure about this one)

Tenneb22 committed:
Rename java packages from screenhandler to menu

Tenneb22 committed:
Rename some resource files and references

Tenneb22 committed:
Rename id, name and description in fabric.mod.json

Tenneb22 committed:
Rename missing directories in the test mod

Tenneb22 committed:
Rename id, name in the test mod fabric.mod.json, and fix the entrypoints and "depends" section

Tenneb22 committed:
Rename and change the rest

Earthcomputer committed:
networking-api-v1 renames (#5058)

Earthcomputer commented:
<img width="4776" height="635" alt="graphviz" src="https://github.com/user-attachments/assets/07e9479d-189b-4687-8848-45b30a30ff8d" />

Here is what is left to do, each module is listed with its dependencies. Don't start on a module until all of its dependencies are complete!

- [x] **fabric-api-base** (N/A)
- [x] **fabric-api-lookup-api-v1** (apple502j): fabric-api-base, fabric-lifecycle-events-v1
- [x] **fabric-biome-api-v1** (apple502j)
- [x] **fabric-block-api-v1** (Cassian)
- [x] **fabric-block-view-api-v2** (Cassian)
- [x] **fabric-client-gametest-api-v1** (Earthcomputer)
- [x] **fabric-command-api-v2** (Toffikk): fabric-api-base
- [x] **fabric-content-registries-v0** (Earthcomputer): fabric-api-base, fabric-lifecycle-events-v1, fabric-resource-loader-v1
- [x] **fabric-convention-tags-v2** (N/A): fabric-api-base, fabric-lifecycle-events-v1
- [x] **fabric-crash-report-info-v1** (coredex-source)
- [x] **fabric-data-attachment-api-v1** (sylv256): fabric-api-base, fabric-entity-events-v1, fabric-networking-api-v1, fabric-object-builder-api-v1
- [x] **fabric-data-generation-api-v1** (Kilip1000): fabric-convention-tags-v2, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-resource-conditions-api-v1, fabric-tag-api-v1
- [x] **fabric-dimensions-v1** (N/A)
- [x] **fabric-entity-events-v1** (Cassian): fabric-api-base
- [x] **fabric-events-interaction-v0** (Cassian): fabric-api-base, fabric-networking-api-v1
- [x] **fabric-game-rule-api-v1** (Tenneb22): fabric-api-base
- [x] **fabric-gametest-api-v1** (apple502j): fabric-api-base, fabric-resource-loader-v1
- [x] **fabric-item-api-v1** (Cassian): fabric-api-base, fabric-resource-loader-v1
- [x] **fabric-item-group-api-v1** (Cassian): fabric-api-base, fabric-resource-loader-v1
- [x] **fabric-key-binding-api-v1** (Cassian)
- [x] **fabric-lifecycle-events-v1** (Earthcomputer): fabric-api-base
- [x] **fabric-loot-api-v3** (CrypticVerse): fabric-api-base, fabric-resource-loader-v1
- [x] **fabric-message-api-v1** (coredex-source): fabric-api-base
- [x] **fabric-model-loading-api-v1**: fabric-api-base, fabric-renderer-api-v1
- [x] **fabric-networking-api-v1** (Earthcomputer): fabric-api-base
- [x] **fabric-object-builder-api-v1** (sylv256): fabric-api-base, fabric-registry-sync-v0, fabric-resource-loader-v1
- [x] **fabric-particles-v1** (sylv256): fabric-api-base, fabric-networking-api-v1
- [x] **fabric-recipe-api-v1** (Kilip1000): fabric-lifecycle-events-v1, fabric-networking-api-v1
- [x] **fabric-registry-sync-v0** (apple502j): fabric-api-base, fabric-networking-api-v1
- [x] **fabric-renderer-api-v1** (sylv256): fabric-api-base, fabric-rendering-v1, fabric-transitive-access-wideners-v1
- [x] **fabric-renderer-indigo** (sylv256): fabric-api-base, fabric-renderer-api-v1, fabric-rendering-v1
- [x] **fabric-rendering-fluids-v1** (deirn): fabric-api-base
- [x] **fabric-rendering-v1** (sylv256): fabric-api-base, fabric-lifecycle-events-v1, fabric-transitive-access-wideners-v1
- [x] **fabric-resource-conditions-api-v1** (N/A)
- [x] **fabric-resource-loader-v1** (apple502j): fabric-api-base
- [x] **fabric-screen-api-v1** (apple502j): fabric-api-base
- [x] **fabric-screen-handler-api-v1** (Tenneb22): fabric-api-base, fabric-networking-api-v1, fabric-registry-sync-v0
- [x] **fabric-serialization-api-v1** (Toffikk)
- [x] **fabric-sound-api-v1** (Toffikk)
- [x] **fabric-tag-api-v1** (Cassian): fabric-api-base, fabric-resource-loader-v1
- [x] **fabric-transfer-api-v1** (deirn): fabric-api-base, fabric-api-lookup-api-v1, fabric-lifecycle-events-v1, fabric-rendering-fluids-v1
- [x] **fabric-transitive-access-wideners-v1** (N/A)

apple502j committed:
api renames, by apple502j (#5057)

* reg sync

* resource loader

* api lookup api

* biome api

cassiancc committed:
Creative Mode Tab API renames (#5059)

* creative mode tab api

Checkstyle

rename buttons

* remove provides

modmuss50 commented on gradle.properties:
```diff
@@ -20,6 +20,7 @@ fabric-client-gametest-api-v1-version=4.3.6
 fabric-command-api-v2-version=2.4.8
 fabric-content-registries-v0-version=10.2.15
 fabric-crash-report-info-v1-version=0.3.24
+fabric-creative-mode-tab-api-v1-version=4.2.37
```
Is this name too verbose? should we drop "mode" from it?

Toffikk replied:
i'd be in favour of dropping it as it is easily understandable what this means without `mode`

cassiancc replied:
I agree on the case of the module name - https://github.com/FabricMC/fabric/pull/5079

Tenneb22 committed:
Merge remote-tracking branch 'origin/api-renames' into api-renames

Tenneb22 committed:
small change

Toffikk committed:
rename: fabric-sound-api-v1

cassiancc committed:
item api

cassiancc committed:
Update CustomDamageTest.java

cassiancc committed:
Update settings in testmod

Toffikk committed:
api renames in serialization api v1

cassiancc committed:
block api

cassiancc committed:
fix renames

Tenneb22 committed:
remove TODOs

cassiancc committed:
tag api

Tenneb22 committed:
rename package and add todo

Earthcomputer committed:
lifecycle-events-v1 renames

Tenneb22 committed:
rename syncId to containerId

cassiancc committed:
Key Mapping API renames

modmuss50 committed:
Merge pull request #5061 from cassiancc/item-api-renames

Block, Block View, Item, and Tag API renames

modmuss50 committed:
Merge pull request #5062 from Toffikk/api/rename-sound-api-v1

api renames in fabric-sound-api-v1

modmuss50 committed:
Merge pull request #5063 from Toffikk/rename/serialization-api-v1

api renames in serialization api v1

modmuss50 committed:
Merge pull request #5064 from Tenneb22/api-renames-menu-api

api rename screen-handler-api to menu-api

modmuss50 committed:
Merge pull request #5065 from Earthcomputer/renames/lifecycle-events-v1

lifecycle-events-v1 renames

cassiancc committed:
Fix mixin config

modmuss50 committed:
Fix build

cassiancc committed:
Merge branch 'api-renames' into keymapping-api

modmuss50 committed:
Merge remote-tracking branch 'origin/26.1' into api-renames

# Conflicts:
#	fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/EquipmentSlotProvider.java
#	fabric-menu-api-v1/src/main/java/net/fabricmc/fabric/api/menu/v1/package-info.java

cassiancc committed:
Merge branch 'api-renames' into keymapping-api

modmuss50 committed:
Fix build again...

Toffikk committed:
renames in command-api v2

Toffikk committed:
Merge remote-tracking branch 'origin/api-renames' into rename/command-api-v2

Toffikk committed:
chore: fix some issues

cassiancc committed:
Update EquipmentSlotProvider.java

Toffikk committed:
chore: fixup javadoc

Toffikk committed:
chore: update some tests’ params

Toffikk committed:
more renames

modmuss50 committed:
Merge pull request #5066 from cassiancc/keymapping-api

Key Mapping API renames

apple502j committed:
game test

apple502j committed:
screen api

coredex-source committed:
fabric-crash-report-info-v1

- Does not seem to need anything else renamed.

coredex-source committed:
fabric-message-api-v1 - commit.1

- ClientReceiveMessageEvents -> ClientLogChatMessageEvents

coredex-source committed:
fabric-message-api-v1 - commit.2

- Patch files previously dependent on ClientReceiveMessageEvents.

deirn committed:
rename rendering-fluids and transfer-api

deirn committed:
run spotlessApply

deirn committed:
where this line comes from?

coredex-source committed:
fabric-message-api-v1 - commit.3

ClientSendMessageEvents -> ClientHandleChatInputEvents

coredex-source committed:
Fix build

coredex-source committed:
fabric-message-api-v1 - commit.4

- Patched ChatListenerMixin.
- Done with client renaming.

coredex-source committed:
fabric-message-api-v1 - commit.5 - done.

- ServerMessageEvents -> ServerChatEvents
- ServerMessageDecoratorEvents -> ServerChatDecoratorEvents
- Patched Mixins.
- Patched testmod.
- Finalized renaming of fabric-message-api-v1.

Tenneb22 committed:
Rename game rule api

Tenneb22 committed:
fix import order issue

Earthcomputer committed:
client-gametest-api-v1 renames

modmuss50 committed:
Merge pull request #5076 from Tenneb22/rename/api-renames-game-rule-api

Rename game-rule-api

modmuss50 committed:
Merge pull request #5071 from apple502j/misc/api-renames-2

Rename GameTest, Screen APIs

modmuss50 committed:
Merge pull request #5074 from deirn/deirn/transfer-rename

Rename rendering-fluids and transfer-api

modmuss50 committed:
Merge pull request #5067 from Toffikk/rename/command-api-v2

renames in command-api v2

modmuss50 committed:
Merge pull request #5077 from Earthcomputer/renames/client-gametest-api-v1

client-gametest-api-v1 renames

cassiancc committed:
Entity Events API renames

cassiancc committed:
Events Interaction API renames

cassiancc committed:
Rename module to Creative Tab API

Earthcomputer committed:
content-registries-v0 renames

CrypticVerse committed:
renames to loot

Kilip1000 committed:
Mapped a bit

(Should've committed more frequently, oops)

Kilip1000 committed:
Mapped a bit

(Should've committed more frequently, oops)

Kilip1000 committed:
A bit more

DennisOchulor commented on fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/event/lifecycle/v1/ServerChunkEvents.java:
```diff
@@ -81,29 +83,29 @@ private ServerChunkEvents() {
-	public static final Event<LevelTypeChange> CHUNK_LEVEL_TYPE_CHANGE = EventFactory.createArrayBacked(LevelTypeChange.class, (world, chunk, oldLevelType, newLevelType) -> { }, callbacks -> (serverWorld, chunk, oldLevelType, newLevelType) -> {
-		for (LevelTypeChange callback : callbacks) {
-			callback.onChunkLevelTypeChange(serverWorld, chunk, oldLevelType, newLevelType);
+	public static final Event<ChunkStatusChange> CHUNK_STATUS_CHANGE = EventFactory.createArrayBacked(ChunkStatusChange.class, (level, chunk, oldChunkStatus, newChunkStatus) -> { }, callbacks -> (serverLevel, chunk, oldChunkStatus, newChunkStatus) -> {
```
Should this event be called FULL_CHUNK_STATUS_CHANGE instead? IIRC ChunkStatus refers to chunk generation stages.

modmuss50 replied:
Yes, now renamed. Thanks.

sylv256 committed:
refactor!(rendering-v1): update API names

sylv256 committed:
fix: checkstyle issues

sylv256 committed:
refactor(rendering-v1)!: `AtlasSourceRegistry` -> `SpriteSourceRegistry`

sylv256 committed:
refactor(object-builder-api-v1)!: api renames

sylv256 committed:
fix: checkstyle

sylv256 committed:
refactor(rendering-v1)!: `ColorProviderRegistry` -> `BlockColorRegistry`

sylv256 committed:
refactor(particles-v1)!: api renames

sylv256 committed:
refactor(particles-v1)!: `ExtendedBlockStateParticleEffectSyncClient` -> `ExtendedBlockParticleOptionSyncClient`

sylv256 committed:
fix: checkstyle lol

coredex-source committed:
Merge branch 'api-renames' into patch/api-renames

Kilip1000 committed:
Mapped some packet-related things

Kilip1000 committed:
Even more packets!

Kilip1000 committed:
client -> minecraft

Kilip1000 committed:
All worlds?

(I hope Intellij didn't mess up at hierarchy renaming)

Kilip1000 committed:
Map the last few things

Also ran the necessary gradle tasks

modmuss50 committed:
Merge pull request #5070 from coredex-source/patch/api-renames

[Patch/Rename] fabric-crash-report-info-v1 && fabric-message-api-v1

modmuss50 committed:
Merge pull request #5078 from cassiancc/entity-events-api-renames

Entity Events and Events Interaction API renames

modmuss50 committed:
Merge pull request #5080 from Earthcomputer/renames/content-registries-v1

content-registries-v0 renames

modmuss50 committed:
Merge pull request #5079 from cassiancc/creative-tab-api

Rename to Creative Tab API

Kilip1000 committed:
Merge branch 'api-renames' into recipe-api-mappings

modmuss50 committed:
Merge pull request #5081 from CrypticVerse/renames/loot-api

Loot API v3 renames

modmuss50 committed:
Merge pull request #5083 from sylv256/api-renames-1

object-builder-api-v1 renames

modmuss50 committed:
Merge pull request #5084 from sylv256/api-renames-2

particles-v1 renames

coredex-source commented:
Mark fabric-dimensions-v1 as done as I do not see anything that requires a rename in there.

Kilip1000 committed:
Merge branch 'api-renames' of https://github.com/FabricMC/fabric into api-renames

sylv256 committed:
refactor(data-attachment-api-v1): api renames

sylv256 committed:
refactor(data-attachment-api-v1): s2c -> clientbound, c2s -> serverbound (the superior names)

sylv256 committed:
refactor(data-attachment-api-v1): packetCodec -> streamCodec

sylv256 committed:
fix: damn checkstyle

sylv256 committed:
Update fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/ChunkSectionLayerMap.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
refactor(rendering-v1): `BlockColorProviderRegistry` -> `ColorProviderRegistry`

sylv256 committed:
refactor(rendering-v1): `AtlasSource` -> `SpriteSource`

sylv256 committed:
refactor(rendering-v1): update names in `LevelRenderEventsTests`

sylv256 committed:
Merge remote-tracking branch 'origin/api-renames' into api-renames

sylv256 committed:
Merge branch 'api-renames' into api-renames

Kilip1000 committed:
Temporary change

Kilip1000 committed:
S2C -> ClientBound

Kilip1000 committed:
Done with ModelProviderMixin (except for writer, which is a bit more complicated)

Kilip1000 committed:
Done with ModelProviderMixin (except for writer, which is a bit more complicated)

Kilip1000 committed:
Done with FabricAdvancementProvider, (except for writer, which is a bit more complicated)

Kilip1000 committed:
Done with FabricAdvancementProvider, (except for writer, which is a bit more complicated)

Kilip1000 committed:
Rename to cachedOutput

Saw cache getting used though, so I'll change this back to cache, oopsy

Kilip1000 committed:
Rename to cachedOutput

Saw cache getting used though, so I'll change this back to cache, oopsy

Kilip1000 committed:
Some more cache things

Kilip1000 committed:
Some more cache things

Kilip1000 committed:
TestModClient things

Kilip1000 committed:
TestModClient things

Kilip1000 committed:
Fixed checkstyle errors?

Kilip1000 committed:
Fixed checkstyle errors?

Kilip1000 committed:
Improve javadoc readability

Kilip1000 committed:
Improve javadoc readability

Kilip1000 committed:
SoundTypeBuilder done?

Kilip1000 committed:
SoundTypeBuilder done?

Kilip1000 committed:
Javadoc misc and registry -> holder

Kilip1000 committed:
Javadoc misc and registry -> holder

Kilip1000 committed:
Misc

Kilip1000 committed:
Misc

Kilip1000 committed:
FabricCodecDataProvider renames

Kilip1000 committed:
FabricCodecDataProvider renames

Kilip1000 committed:
FabricDynamicRegistryProvider - 1 (not done)

Maybe did something wrong. This is a quite convoluted commit.

Kilip1000 committed:
FabricDynamicRegistryProvider - 1 (not done)

Maybe did something wrong. This is a quite convoluted commit.

Kilip1000 committed:
FabricDynamicRegistryProvider - 2 (not done)

Kilip1000 committed:
FabricDynamicRegistryProvider - 2 (not done)

Kilip1000 committed:
entries -> ressources

Is this good?

Kilip1000 committed:
entries -> ressources

Is this good?

Earthcomputer committed:
Rename ClientBound -> Clientbound

Kilip1000 committed:
Merge pull request #1 from Earthcomputer/recipe-api-mappings

Rename ClientBound -> Clientbound

Kilip1000 committed:
RegistryLookup

Kilip1000 committed:
RegistryLookup

Kilip1000 committed:
registryLookup

Kilip1000 committed:
registryLookup

Kilip1000 committed:
More of the same

Kilip1000 committed:
More of the same

sylv256 commented:
Hey y'all, we missed `FabricBlockView` in `blockview-api-v2`

sylv256 committed:
refactor(renderer-api-v1)!: api renames

modmuss50 committed:
Merge pull request #5082 from sylv256/api-renames

rendering-v1 renames

modmuss50 committed:
Merge pull request #5086 from sylv256/api-renames-1

data-attachment-api-v1 renames

sylv256 committed:
Merge branch 'api-renames' into api-renames-2

sylv256 committed:
refactor(renderer-api-v1): `TextureAtlasMixinAtlas` -> `TextureAtlasMixin`

sylv256 committed:
refactor(renderer-api-v1): `blockAndTintGetter` -> `level`

sylv256 committed:
Merge remote-tracking branch 'origin/api-renames-2' into api-renames-2

sylv256 committed:
Update fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/model/ModelStateHelper.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
Update fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/render/FabricBlockRenderDispatcher.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
Update fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/render/RenderLayerHelper.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
Update fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/render/RenderLayerHelper.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
Update fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/mixin/renderer/client/block/render/BlockFeatureRendererMixin.java

Co-authored-by: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>

sylv256 committed:
refactor(renderer-api-v1)!: `FabricBlockStateModel#particleSprite` -> `#particleIcon`

sylv256 committed:
refactor(renderer-api-v1)!: `ExtendedBlockStateModelSubmit` -> `ExtendedBlockModelSubmit`

sylv256 committed:
fix(renderer-api-v1): `#particleSprite` -> `#particleIcon`

sylv256 committed:
refactor(renderer-api-v1)!: `FabricBlockModelShaper#getModelParticleSprite` -> `#getModelParticleIcon`

sylv256 committed:
refactor(renderer-api-v1)!: `FabricBlockModelShaper#getModelParticleIcon` -> `#getParticleIcon`

sylv256 committed:
refactor(renderer-api-v1)!: missed api renames

sylv256 committed:
refactor(renderer-indigo): api renames

sylv256 committed:
refactor(renderer-indigo): missed renames

modmuss50 committed:
Merge pull request #5089 from sylv256/api-renames-2

renderer-api-v1 renames

Kilip1000 committed:
Merge branch 'data-generation-api-renames' of https://github.com/Kilip1000/fabric into data-generation-api-renames

Kilip1000 committed:
Missed one

Kilip1000 committed:
Params

Kilip1000 committed:
Another mixin rename

Missed a comment

Revert

This time?

Rename?

Why doesn't rename the interface that's getting injected work?

Revert "Rename?"

This reverts commit 1a1b9814a92e9d3780fe1f274ad78effa9bcfa12.

Kilip1000 committed:
Revert "Another mixin rename"

This reverts commit 89867543715ad91a9bf3d88d66d665523d8565ec.

Kilip1000 committed:
I keep forgetting this one

Kilip1000 committed:
Data -> Pack

Kilip1000 committed:
Data -> Pack 2?

Kilip1000 committed:
Generators with an s, also block, not blockState

Kilip1000 committed:
Missed one

Kilip1000 committed:
I think I was a bit too bold with that one

Kilip1000 committed:
registry entry -> holder in javadoc

This one is debatable?

Kilip1000 committed:
Data -> Pack 3

Kilip1000 committed:
Interface injection please work?

Kilip1000 committed:
BlockLootTableGenerator -> BlockLootSubProvider

Kilip1000 committed:
Revert "Interface injection please work?"

This reverts commit 34febbcd0c471fcf0931fa7726fd2a3d03053dd2.

sylv256 committed:
Merge branch 'api-renames' into api-renames

Kilip1000 committed:
Data -> Pack 4

Kilip1000 committed:
Undo an oopsy

Kilip1000 committed:
These parameter names should go unchanged

Went through the git diff and undid some silliness manually

Kilip1000 committed:
Undid even more unnecessary changes

Kilip1000 committed:
I think I went to overboard with my undos, same bug as before

Kilip1000 committed:
Some more undos

Kilip1000 committed:
More undos

Kilip1000 committed:
Undos, also, accidentally based this branch on the wrong branch

Kilip1000 committed:
Undos

modmuss50 committed:
Merge pull request #5085 from Kilip1000/recipe-api-mappings

fabric-recipe-api-v1 renames

Kilip1000 committed:
I could at least change the javadoc..

Kilip1000 committed:
yep, undid this one too

sylv256 committed:
Merge branch 'FabricMC:api-renames' into api-renames

sylv256 committed:
refactor(model-loading-api-v1)!: api renames

modmuss50 committed:
Merge pull request #5090 from sylv256/api-renames

renderer-indigo renames

modmuss50 committed:
Merge pull request #5093 from sylv256/api-renames-1

model-loading-api-v1 renames

Kilip1000 committed:
Now only one thing is left

Kilip1000 committed:
oops, forgot mcdev

Kilip1000 committed:
Fix

modmuss50 committed:
Merge pull request #5088 from Kilip1000/data-generation-api-renames

fabric-data-generation-api-v1 renames

modmuss50 committed:
Merge remote-tracking branch 'origin/26.1' into api-renames

# Conflicts:
#	fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/mixin/entity/event/LivingEntityMixin.java
#	gradle.properties

Earthcomputer committed:
World -> Level

Earthcomputer committed:
tickCounter -> deltaTracker

Earthcomputer committed:
drawContext -> graphics

Earthcomputer committed:
Go through vibecoded script output

Earthcomputer committed:
Interface injections and spotless

Earthcomputer committed:
Text -> component

Earthcomputer committed:
Merge branch 'api-renames' into renames/search

# Conflicts:
#	fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricLanguageProvider.java

Earthcomputer committed:
Fix mixin config

Earthcomputer committed:
Rename screenshot template

Earthcomputer committed:
Address review

modmuss50 committed:
Merge pull request #5095 from Earthcomputer/renames/search

Renames/search

modmuss50 committed:
Revert rename of PlayerInventoryStorage

modmuss50 commented on fabric-message-api-v1/src/client/java/net/fabricmc/fabric/api/client/message/v1/ClientLogChatMessageEvents.java:
```diff
@@ -32,8 +32,8 @@
  */
-public final class ClientReceiveMessageEvents {
-	private ClientReceiveMessageEvents() {
+public final class ClientLogChatMessageEvents {
```
Why?

coredex-source replied:
Yarn: net.minecraft.client.session.report.log.ReceivedMessage
Mojang: net.minecraft.client.multiplayer.chat.LoggedChatMessage

modmuss50 replied:
I dont think this was based on the yarn name tbh. Its more of a case of the client recieving the chat message, with the new name to me it feels like its related to logging the chat message to the log file.

coredex-source replied:
That does make sense yeah. Can be rolled back I suppose.
My bad, I just tried to be as consistent with mojang as possible.

modmuss50 replied:
Its not a problem at all, its good to look at this stuff in detail.

kevinthegreat1 replied:
Hi, I am the author of this api. I don't think this rename makes sense at all. The new name does not indicated clearly that these events are used for receiving messages. I think keeping the original name is a better option.

coredex-source replied:
I agree.

modmuss50 commented on fabric-message-api-v1/src/client/java/net/fabricmc/fabric/api/client/message/v1/ClientLogChatMessageEvents.java:
```diff
@@ -44,11 +44,11 @@ private ClientReceiveMessageEvents() {
 	 * {@link #CHAT_CANCELED} will be triggered instead of {@link #CHAT}.
 	 */
-	public static final Event<AllowChat> ALLOW_CHAT = EventFactory.createArrayBacked(AllowChat.class, listeners -> (message, signedMessage, sender, params, receptionTimestamp) -> {
+	public static final Event<IsChatAllowed> IS_CHAT_ALLOWED = EventFactory.createArrayBacked(IsChatAllowed.class, listeners -> (message, playerChatMessage, sender, bind, timeStamp) -> {
```
Why was this renamed?

coredex-source replied:
Mojang: Minecraft.ChatStatus.isChatAllowed
Yarn: MinecraftClient.ChatRestriction.allowsChat

dexman545 replied:
For the event, `ALLOW_CHAT` makes more sense

sylv256 replied:
The point of the PR is to be consistent with Mojang's names.

coredex-source replied:
Yeah I tried to look through everything I could before renaming.

cputnam-a11y replied:
> The point of the PR is to be consistent with Mojang's names.

Yes, but I would say that this doesn't count for names that weren't weren't derived from the yarn name, but rather from fapi's contributing.md solely. Obviously there will be exceptions where contributing.md derived from yarn, but imo this isn't one of them. In fapi all events that deal with cancelation typically use the AllowX pattern

Juuxel replied:
Yeah, all of these events in Fabric API are called `ALLOW_XYZ`. It's not really a Yarn thing as they usually don't match any method in the game's code.

modmuss50 commented on fabric-message-api-v1/src/client/java/net/fabricmc/fabric/api/client/message/v1/ClientLogChatMessageEvents.java:
```diff
@@ -98,15 +98,15 @@ private ClientReceiveMessageEvents() {
+	public static final Event<Chat> CHAT = EventFactory.createArrayBacked(Chat.class, listeners -> (message, playerChatMessage, sender, bind, timeStamp) -> {
 		for (Chat listener : listeners) {
-			listener.onReceiveChatMessage(message, signedMessage, sender, params, receptionTimestamp);
+			listener.onLogChatMessage(message, playerChatMessage, sender, bind, timeStamp);
```
Why?

coredex-source replied:
Yarn: net.minecraft.client.session.report.log.ReceivedMessage
Mojang: net.minecraft.client.multiplayer.chat.LoggedChatMessage

modmuss50 commented:
I have updated the PR description with an auto generated list of the renames.

kevinthegreat1 commented on fabric-message-api-v1/src/client/java/net/fabricmc/fabric/api/client/message/v1/ClientHandleChatInputEvents.java:
```diff
@@ -22,8 +22,8 @@
-public final class ClientSendMessageEvents {
-	private ClientSendMessageEvents() {
+public final class ClientHandleChatInputEvents {
+	private ClientHandleChatInputEvents() {
```
Same as my comment on the receive message events. I also authored this, and I don't think this rename makes any sense. This does not clearly indicate that these events are for sending messages. I think the original name is way better in this case.

kevinthegreat1 replied:
Chat input could easily be mistaken for events such as a character typed rather than sending a message or command.

Additionally, the two send and receive events are counterparts, but this rename no longer indicates that.

modmuss50 committed:
Checkstyle fixes

modmuss50 committed:
Name adjustments

modmuss50 commented:
PR updated with the chat name changes, and reverted to `generateBlockStateModels`.

Please review the changes and question anything that doesnt seem right.

modmuss50 committed:
CHUNK_STATUS_CHANGE -> FULL_CHUNK_STATUS_CHANGE

Juuxel commented on fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/impl/content/registry/CompostableRegistryRegistryImpl.java:
```diff
@@ -21,21 +21,21 @@
+import net.fabricmc.fabric.api.registry.CompostableRegistry;
 
-public class CompostingChanceRegistryImpl implements CompostingChanceRegistry {
+public class CompostableRegistryRegistryImpl implements CompostableRegistry {
```
"Registry registry"?

Juuxel commented on fabric-creative-tab-api-v1/src/client/java/net/fabricmc/fabric/api/client/creativemodetab/v1/FabricCreativeModeInventoryScreen.java:
```diff
@@ -69,21 +69,21 @@ default int getPageCount() {
+	 * Returns an ordered list containing the creative mode tabs on the requested page.
 	 */
-	default List<CreativeModeTab> getItemGroupsOnPage(int page) {
+	default List<CreativeModeTab> getCreativeModeTabsOnPage(int page) {
```
The new names of these methods sound quite verbose. `getTabsOnPage` etc. are clear enough IMO without being as wordy.

modmuss50 replied:
Agreed 👍

Juuxel commented on checkstyle.xml:
```diff
@@ -197,7 +197,7 @@
 			-->
 			<property name="format"
-			value="^net\.fabricmc\.fabric\.(api(?!\.common\.)(\.client|\.server|)(\.(?!client\.|server\.)[a-z]+([a-rt-z]|ss))+\.v[0-9]+|(impl|mixin|test)(?!\.common\.)(\.client|\.server|)(\.(?!client\.|server\.)[a-z]+([a-rt-z]|ss))+(\.v[0-9]+)?|api\.(event|util|biomes\.v1|registry|client\.screen|container|block|entity|client\.itemgroup|client\.keybinding|tag|tools|client\.model|network|server|client\.rend ...
+			value="^net\.fabricmc\.fabric\.(api(?!\.common\.)(\.client|\.server|)(\.(?!client\.|server\.)[a-z]+([a-rt-z]|ss))+\.v[0-9]+|(impl|mixin|test)(?!\.common\.)(\.client|\.server|)(\.(?!client\.|server\.)[a-z]+([a-rt-z]|ss))+(\.v[0-9]+)?|api\.(event|util|biomes\.v1|registry|client\.screen|container|block|entity|client\.creativemodetab|client\.keymapping|tag|tools|client\.model|network|server|client ...
```
Can't we fix these legacy packages while we're at it instead of renaming them to use the same broken format?

(Edit: I mean specifically those ones that were modified in this PR. The rest are fine since they existed beforehand, but we shouldn't add new ones.)

modmuss50 replied:
A lot of these are no longer needed I have removed them. I'm unsure if we want to fix all of these as it really will affect everything (events need moving). Im tempted to say no.

modmuss50 replied:
As discussed, I dont think we are going to fix this. Especially not in this PR.

Juuxel commented on fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommandsRegistrationCallback.java:
```diff
@@ -28,20 +28,20 @@
-	Event<ClientCommandRegistrationCallback> EVENT = EventFactory.createArrayBacked(ClientCommandRegistrationCallback.class, (callbacks) -> (dispatcher, registryAccess) -> {
-		for (ClientCommandRegistrationCallback callback : callbacks) {
-			callback.register(dispatcher, registryAccess);
+public interface ClientCommandsRegistrationCallback {
```
This rename is nonsensical and unnecessary considering there was no Yarn name for this concept to begin with.

modmuss50 replied:
Reverted

modmuss50 committed:
ClientCommandsRegistrationCallback -> ClientCommandRegistrationCallback
getCreativeModeTabsOnPage -> getTabsOnPage
getSelectedCreativeModeTab -> getSelectedTab
setSelectedCreativeModeTab -> setSelectedTab

Juuxel commented on fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommands.java:
```diff
@@ -53,19 +53,19 @@
  * <pre>
  * {@code
- * ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> {
+ * ClientCommandsRegistrationCallback.EVENT.register((dispatcher, buildContext) -> {
```
```suggestion
 * ClientCommandRegistrationCallback.EVENT.register((dispatcher, buildContext) -> {
```

modmuss50 committed:
Remove some old package regex hacks

modmuss50 committed:
Update fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommands.java

Co-authored-by: Juuz <6596629+Juuxel@users.noreply.github.com>

deirn commented on fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/package-info.java:
```diff
@@ -59,13 +59,13 @@
- * {@link net.fabricmc.fabric.api.transfer.v1.item.PlayerInventoryStorage PlayerInventoryStorage} when interaction with
- * {@code Inventory}-based APIs is required.
+ * {@link net.fabricmc.fabric.api.transfer.v1.item.ContainerStorage ContainerStorage} or
+ * {@link net.fabricmc.fabric.api.transfer.v1.item.PlayerInventoryStorage InventoryStorage} when interaction with
```
```suggestion
 * {@link net.fabricmc.fabric.api.transfer.v1.item.PlayerInventoryStorage PlayerInventoryStorage} when interaction with
```

deirn commented on fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/impl/transfer/item/ContainerStorageImpl.java:
```diff
@@ -91,35 +91,35 @@ public List<SingleSlotStorage<ItemVariant>> getSlots() {
 	@Override
 	public String toString() {
-		return "InventoryStorage[" + DebugMessages.forInventory(inventory) + "]";
+		return "InventoryStorage[" + DebugMessages.forInventory(container) + "]";
```
```suggestion
		return "ContainerStorage[" + DebugMessages.forInventory(container) + "]";
```

deirn commented on fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/impl/transfer/item/ContainerStorageImpl.java:
```diff
@@ -29,56 +29,56 @@
 	 * Global wrapper concurrent map.
 	 *
 	 * <p>A note on GC: weak keys alone are not suitable as the InventoryStorage slots strongly reference the Inventory keys.
 	 * Weak values are suitable, but we have to ensure that the InventoryStorageImpl remains strongly reachable as long as
```
```suggestion
	 * <p>A note on GC: weak keys alone are not suitable as the ContainerStorage slots strongly reference the Inventory keys.
	 * Weak values are suitable, but we have to ensure that the ContainerStorageImpl remains strongly reachable as long as
```

Toffikk committed:
rename access to buildContext in sound api-v1 testmod

modmuss50 committed:
Merge pull request #5099 from Toffikk/patch-1

rename access to buildContext in sound api-v1 testmod

DennisOchulor committed:
Renames for FULL_CHUNK_STATUS_CHANGE

modmuss50 committed:
Merge pull request #5100 from DennisOchulor/full-chunk-status-renames

Renames for FULL_CHUNK_STATUS_CHANGE

modmuss50 committed:
Update fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/package-info.java

Co-authored-by: D. Firmansyah <deirn@bai.lol>

modmuss50 committed:
Update fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/impl/transfer/item/ContainerStorageImpl.java

Co-authored-by: D. Firmansyah <deirn@bai.lol>

modmuss50 committed:
Update fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/impl/transfer/item/ContainerStorageImpl.java

Co-authored-by: D. Firmansyah <deirn@bai.lol>

cassiancc committed:
Further rename Creative Tab API

modmuss50 committed:
Merge pull request #5106 from cassiancc/creative-tab-renames

Further creative tab API renames

modmuss50 committed:
Name tweaks

modmuss50 committed:
Use DataComponentHolder and TypedInstance for TransferVariant's

modmuss50 committed:
HOLDER_LOOKUP_KEY -> REGISTRY_LOOKUP_KEY

modmuss50 committed:
accept -> insert

modmuss50 commented:
PR description has been updated, please take a look as I plan to merge this tomorrow or Wednesday.
